### PR TITLE
add disable editor

### PR DESCRIPTION
**Pull request by lvenier**

### CODE REVIEW

File: settings_dev.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ module.exports = {
         user:"admin",pass:"$2a$08$s.NFdSn4Gm4d7gHErya//e6O8RO1/3f7TZ7zflXJ9jfFV0cI6jGwK"
     },
     uiPort: process.env.PORT || 1880,
+    disableEditor: process.env.DISABLE_EDITOR || false,
```
Comment (fanshan) [inline settings_dev.js:24]: Why not `true` for development settins?

**Review by fanshan — CHANGES_REQUESTED** [review]

File: settings.js [hunk 2]
```diff
@@ -22,6 +22,7 @@ module.exports = {
         pass: process.env.API_PASSWORD || "$2a$08$s.NFdSn4Gm4d7gHErya//e6O8RO1/3f7TZ7zflXJ9jfFV0cI6jGwK"
     },
     uiPort: process.env.PORT || 1880,
+    disableEditor: process.env.DISABLE_EDITOR || false,
```
Comment (fanshan) [inline settings.js:25]: Can we disable the editor except in the dev mode?

Comment (lvenier) [reply settings.js:25]: why not ! but the issue is that will change the current behavior and will require a major change I guess. For upgrading the env var must be added in the running instance...

File: settings_dev.js [hunk 3]
```diff
@@ -21,6 +21,7 @@ module.exports = {
         user:"admin",pass:"$2a$08$s.NFdSn4Gm4d7gHErya//e6O8RO1/3f7TZ7zflXJ9jfFV0cI6jGwK"
     },
```
[the remark this answers is not in the document]
Comment (lvenier) [reply settings_dev.js:22]: Agree ! but that would be part of an other PR as this one is for disabling editor toggler

File: settings.js (hunk 2 above)
Comment (fanshan) [reply settings.js:25]: This issue cost me a flash on the road.

**Review by fanshan — APPROVED** [review]

**Pull request merged**